rpcs3/Emu/System.h

```diff
@@ -216,6 +216,7 @@ class Emulator final
 	std::vector<u8> data;
 	std::vector<u128> klic;
 	std::string disc;
+	std::string inherited_ps3_game;
```

Comment:
Why not call it
m_inherited_ps3_game
And put it to the other private members?

It seems out of place here.